app/queries/payments_requiring_action.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class PaymentsRequiringAction
+  def initialize(user)
+    @user = user
+  end
+
+  attr_reader :user
```

Comment:
I'd rather make this private. The smaller this class's public API the more changeable it'll be.